include/mgp.hpp

```diff
@@ -3185,6 +3187,55 @@ inline bool Value::IsDuration() const { return mgp::value_is_duration(ptr_); }
 inline bool Value::operator==(const Value &other) const { return util::ValuesEqual(ptr_, other.ptr_); }
 
 inline bool Value::operator!=(const Value &other) const { return !(*this == other); }
+
+inline bool Value::operator<(const Value &other) const {
+  const mgp::Type &type = Type();
+  if (type != other.Type() && !(IsNumeric() && other.IsNumeric())) {
+    throw ValueException("Values have to be of the same type");
+  }
+
+  switch (type) {
+    case Type::Null:
+      throw ValueException("Cannot compare Null types");
+      break;
+    case Type::Bool:
+      return ValueBool() < other.ValueBool();
+      break;
+    case Type::Int:
+      return ValueNumeric() < other.ValueNumeric();
+      break;
+    case Type::Double:
+      return ValueNumeric() < other.ValueNumeric();
+      break;
+    case Type::String:
+      return ValueString() < other.ValueString();
+      break;
+    case Type::Node:
+      return ValueNode() < other.ValueNode();
+      break;
+    case Type::Relationship:
+      return ValueRelationship() < other.ValueRelationship();
+      break;
+    case Type::Date:
+      return ValueDate() < other.ValueDate();
+      break;
+    case Type::LocalTime:
+      return ValueLocalTime() < other.ValueLocalTime();
+      break;
+    case Type::LocalDateTime:
+      return ValueLocalDateTime() < other.ValueLocalDateTime();
+      break;
+    case Type::Duration:
+      return ValueDuration() < other.ValueDuration();
+      break;
+    default:
+      if (IsPath() || IsList() || IsMap()) {
+        throw ValueException("Operator < is not defined for this data type");
+      } else {
+        return false;
+      }
```

Comment:
Why do you need this else?

Reply:
I left it in case someone tries something I oversighted or is undefined in compariosn to return false, but will remove it for now, there shouldnt be undefined behaviour

Reply:
And without it I get error, because its a non-void function that has to return something, so i will remove false and rasie exception